internal/pkg/cli/job_logs.go

```diff
@@ -198,54 +205,58 @@ func (o *jobLogsOpts) getTargetEnv() (*config.Environment, error) {
 	return o.targetEnv, nil
 }
 
-// Execute outputs logs of the job.
-func (o *jobLogsOpts) Execute() error {
-	if err := o.initLogsSvc(); err != nil {
+func (o *jobLogsOpts) validateOrAskApp() error {
+	if o.appName != "" {
+		_, err := o.configStore.GetApplication(o.appName)
 		return err
 	}
-	eventsWriter := logging.WriteHumanLogs
-	if o.shouldOutputJSON {
-		eventsWriter = logging.WriteJSONLogs
+	app, err := o.sel.Application(jobAppNamePrompt, wkldAppNameHelpPrompt)
+	if err != nil {
+		return fmt.Errorf("select application: %w", err)
 	}
+	o.appName = app
+	return nil
+}
 
-	var limit *int64
-	if o.limit != 0 {
-		limit = aws.Int64(int64(o.limit))
+func (o *jobLogsOpts) validateAndAskJobEnvName() error {
+	if o.envName != "" {
+		if _, err := o.getTargetEnv(); err != nil {
+			return err
+		}
 	}
-	err := o.logsSvc.WriteLogEvents(logging.WriteLogEventsOpts{
-		Follow:    o.follow,
-		Limit:     limit,
-		EndTime:   o.endTime,
-		StartTime: o.startTime,
-		TaskIDs:   o.taskIDs,
-		OnEvents:  eventsWriter,
-	})
+	if o.name != "" {
+		if _, err := o.configStore.GetJob(o.appName, o.name); err != nil {
+			return err
+		}
+	}
+	deployedJob, err := o.sel.DeployedJob(jobLogNamePrompt, jobLogNameHelpPrompt, o.appName, selector.WithEnv(o.envName), selector.WithName(o.name))
 	if err != nil {
-		return fmt.Errorf("write log events for job %s: %w", o.name, err)
+		return fmt.Errorf("select deployed jobs for application %s: %w", o.appName, err)
 	}
+	o.name = deployedJob.Name
+	o.envName = deployedJob.Env
 	return nil
 }
 
 // buildJobLogsCmd builds the command for displaying job logs in an application.
 func buildJobLogsCmd() *cobra.Command {
 	vars := jobLogsVars{}
 	cmd := &cobra.Command{
-		Use:    "logs",
-		Short:  "Displays logs of a deployed job.",
-		Hidden: true,
+		Use:   "logs",
+		Short: "Displays logs of a deployed job.",
 		Example: `
   Displays logs of the job "my-job" in environment "test".
   /code $ copilot job logs -n my-job -e test
   Displays logs in the last hour.
   /code $ copilot job logs --since 1h
-  Displays logs from 2006-01-02T15:04:05 to 2006-01-02T15:05:05.
-  /code $ copilot job logs --start-time 2006-01-02T15:04:05+00:00 --end-time 2006-01-02T15:05:05+00:00
-Displays logs from specific task IDs.
+  Displays logs from the last execution of the job.
+  /code $ copilot job logs --last 1
+  Displays logs from specific task IDs.
   /code $ copilot job logs --tasks 709c7eae05f947f6861b150372ddc443,1de57fd63c6a4920ac416d02add891b9
```

Comment:
nit: maybe we can update this example as 
```suggestion
/code $ copilot job logs --tasks 709c7ea,1de57fd
```
I took this from the doc - it looks cleaner and is consistent with the doc!